src/system/system.gni

```diff
@@ -37,7 +37,10 @@ declare_args() {
 }
 
 declare_args() {
-  # use the dispatch library
+  # do not use libev by default
+  chip_system_config_use_libev = false
```

Comment:
Could you add a variant to enable libev usage in `scripts/build/builders/host.py` ?
That would make it easier to build and test this and potentially also use it in our CI builds if we need to maintain its compilation ability.

Reply:
I'd need some guidance/hints how to do that.

I managed to understand the single path through the gn build needed to use chip as a SDK submodule from an external project, but am not familiar enough yet with all the build and CI automation around that.

I guess it would be something like [connectedhomeip-15-51-02.patch](https://github.com/project-chip/connectedhomeip/files/10353069/connectedhomeip-15-51-02.patch), but I have not enough insight to understand the context this would need to integrate with 😟

Reply:
spun off into new issue #27022